tasks/loop-mocha.js

```diff
@@ -33,7 +33,7 @@ module.exports = function (grunt) {
       otherOptions = {},
       reportLocation = loopOptions.reportLocation || '',
       asyncMethod = (loopOptions.parallel && loopOptions.parallel !== false) ? "map" : "mapSeries",
-      binPath = '.bin/mocha' + (process.platform === 'win32' ? '.cmd' : ''),
+      binPath = (options.mocha_bin || '.bin/mocha') + (process.platform === 'win32' ? '.cmd' : ''),
```

Comment:
I wonder, if we are specifying a `mocha_bin` explicitly, if we would use the entire value and not add `.cmd` for the `win32` case?